main/res/values/strings.xml

```diff
@@ -1025,6 +1029,8 @@
     <string name="waypoint_duplicate">Duplicate waypoint</string>
     <string name="waypoint_duplicated">Waypoint duplicated</string>
     <string name="waypoint_copy_of">Copy of</string>
+    <string name="waypointcalc_notes">Notes:</string>
```

Comment:
Thanks for the screenshot.

"Resault" -> "Result"

And I'm still not convinced the ":" is useful or necessary, both for "Result" and "Notes".